op-signer/tls-server.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+# ------------------------------------------------------------------------------
+# Usage:
+# export SIGNER_SERVER_HOST=<DNS or IP of op-signer server>
```

Comment:
what is the purpose of specifying the server ip?

Reply:
The server IP is the server's identity during TLS handshakes, which enables clients using IP to access the server securely.